spice/utils.py

```diff
@@ -22,3 +22,7 @@ def transcription_request_cost(model: TranscriptionModel, input_length: float) -
         return (model.input_cost * input_length) / 100
     else:
         return None
+
+
+def print_stream(text: str) -> None:
```

Comment:
Is this just a general utility function for people to use with the library? I think thats a good idea just making sure it wasn't accidentally left in

Reply:
Yeah, see my example. I think it makes code that really wants the whole response before it continues but also wants to stream the response for UX a lot cleaner.